### PR TITLE
Deploy template project to template k8s namespace

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
 # hmpps-template-typescript
 Template github repo used for new Typescript based projects.
 
+# Instructions
+
+If this is a Digital Prison Services project then the project will be created as part of bootstrapping - 
+see https://github.com/ministryofjustice/dps-project-bootstrap.
```
Comment (petergphillips) [inline README.md:7]: isn't this project now used by lots of probation people too?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+
+When deploying to a new namespace, you may wish to use this template typescript project namespace as the basis for your new namespace:
+
+<https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-template-typescript>
+
+This template namespace includes an AWS elasticache setup - which is required by this template project.
+
+Copy this folder, update all the existing namespace references, and submit a PR to the CloudPlatform team. Further instructions from the CloudPlatform team can be found here: <https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide>
+
+## Renaming from HMPPS Template Kotlin - github Actions
```
Comment (petergphillips) [inline README.md:19]: reference to kotlin here - this is the typescript project

Comment (andrewrlee) [inline README.md:19]: reference to kotlin

this doesn't have github actions enabled

Comment (mattops) [reply README.md:19]: thanks, fix coming.

File: README.md (hunk 1 above)
Comment (mattops) [reply README.md:7]: updated.

File: README.md (hunk 2 above)
Comment (mattops) [reply README.md]: fixed

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**